rosdep/base.yaml

```diff
@@ -2826,6 +2826,12 @@ libopal-dev:
   fedora: [opal-devel]
   gentoo: [net-libs/opal]
   ubuntu: [libopal-dev]
+libopenal-dev:
+  arch: [openal]
+  debian: [libopenal-dev]
+  fedora: [openal-soft]
```

Comment:
For Fedora, I think you want `openal-soft-devel` to get the development packages.